### DISCUSSION TITLE
What happened to @react-spectrum/splitview ?

### DISCUSSION THREAD

**Discussion by cj** [adobe/react-spectrum#1978] [Q&A] [answered]
Hi All!

What happened to @react-spectrum/splitview https://github.com/adobe/react-spectrum/tree/main/packages/%40react-spectrum/splitview ?

Should I use https://opensource.adobe.com/spectrum-web-components/components/split-view instead?

**Comment by snowystinger** [comment, marked as answer]
Hey, that component was deprioritized and it hasn’t made it back into our queue yet to finish. We're also not entirely sure that Design will want this component in particular, but I imagine something like it will come in the future. See components we aim to support https://spectrum.adobe.com/
